Tools/Scripts/webkitdirs.pm

```diff
@@ -917,6 +917,8 @@ sub availableXcodeSDKs
 
 sub isValidXcodeSDKPlatformName($) {
     my $name = shift;
+    # strip sdk version and variant qualifiers prior to validating platform
```

Comment:
This comment says what the code does, but not why. I do not immediately understand why - `make` always works with the active Xcode anyway, so it seems like there would be no need to specify SDK version, as there's only one.